ui/src/testing/stubs/workspace-service-stub.ts

```diff
@@ -155,28 +154,26 @@ export class WorkspacesServiceStub {
         observer.complete();
       }, 0);
     });
-    return observable;
   }
 
-  public shareWorkspace(workspaceNamespace: string,
-      workspaceId: string,
-      request: ShareWorkspaceRequest): Observable<ShareWorkspaceResponse> {
-        const observable = new Observable(observer => {
-          setTimeout(() => {
-            const updateIndex = this.workspaces.findIndex(function(workspace: Workspace) {
-              if (workspace.id === workspaceId) {
-                return true;
-              }
-            });
-            if (updateIndex === -1) {
-              observer.error(new Error(`Error sharing. Workspace with `
-                                      + `id: ${workspaceId} does not exist.`));
-            }
-            this.workspaces[updateIndex].userRoles = request.items;
-            observer.next({});
-            observer.complete();
-          }, 0);
+  shareWorkspace(workspaceNamespace: string,
+                 workspaceId: string,
+                 request: ShareWorkspaceRequest): Observable<ShareWorkspaceResponse> {
+      return new Observable<ShareWorkspaceResponse>(observer => {
+      setTimeout(() => {
+        const updateIndex = this.workspaces.findIndex(function(workspace: Workspace) {
+          if (workspace.id === workspaceId) {
+            return true;
+          }
         });
-        return observable;
+        if (updateIndex === -1) {
+          observer.error(new Error(`Error sharing. Workspace with `
+                                  + `id: ${workspaceId} does not exist.`));
```

Comment:
Nit: above, you converted this from lined up (like this) to an indent.

Reply:
Also good catch - the above change was a mistake